app/templates/ratemypdf.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>

```

Comment:
I'm having trouble figuring out what it is, but the page isn't super mobile friendly; the header is still really small, and the three boxes, which I'd expect to resize and only show one per row, still are in one row, and the text in the boxes is almost too small to read. ![image](https://user-images.githubusercontent.com/6252212/231206210-7f33c979-f331-44a5-80a9-acc22b27e33e.png)